### ISSUE TITLE
warning MSB3030: Could not copy the file "ApplicationInsights.config" because it was not found.

### ISSUE THREAD

**Issue by severisv** [microsoft/ApplicationInsights-dotnet#1085]
After upgrading to netcoreapp2.2 and upgrading Microsoft.ApplicationInsights.AspNetCore from 2.5 to 2.6.1 I'm getting this compilation warning:

```C:\Users\me\.nuget\packages\microsoft.applicationinsights.windowsserver\2.9.1\build\Microsoft.ApplicationInsights.WindowsServer.targets(9,5): warning MSB3030: Could not copy the file "ApplicationInsights.config" because it was not found.```

I'm using AI like this:

```
var host = new WebHostBuilder()
                ...Add lots of different stuff
                .UseStartup<Startup>()
                .UseApplicationInsights()
                .Build();
```

## Version Info 
SDK Version : 2.2.100
netcoreapp2.2
Windows 10

**Comment by SergeyKanzhelev** [comment]
This should also be fixed: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/86af02284d4916da947bb9fd3a44e46cf42845ea/src/ServerTelemetryChannel/Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.targets#L8

**Comment by SergeyKanzhelev** [comment]
@MS-TimothyMothra can you please ACK that this will make it to the closest next release. Looks very critical

**Comment by biznearme** [comment]
I ran into this issue today.  I created an empty file ApplicationInsights.config and placed it in the root folder of the project.  Project compiles with no errors now.

**Comment by MS-TimothyMothra** [comment]
@biznearme Can you confirm which SDK version you experienced this?

**Comment by biznearme** [comment]
Microsoft.NetCore.App v2.2.0 and Microsoft.ApplicationInsights.AspNetCore v2.6.1

**Comment by cijothomas** [comment]
@MS-TimothyMothra is this fixed?

**Comment by MS-TimothyMothra** [comment]
@cijothomas I haven't had any time to look into this.

**Comment by cijothomas** [comment]
@TimothyMothra reminder to look at this before we ship stable.

**Comment by TimothyMothra** [comment]
I'm reviewing this today and I haven't been able to reproduce this issue.

I worked on a fix for this back in February.
All the versions that were reported with this bug predate those fixes (released in AspNetCore v2.7).
I'm going to assume this is fixed now and close this issue.

If the author is still having issues, please comment and feel free to re-open.
